packages/e2e-tests/src/specs/editor/fontCheck/fontCheckMetrics.js

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  createNewStory,
+  insertStoryTitle,
+  publishStory,
+  addRequestInterception,
+} from '@web-stories-wp/e2e-test-utils';
+
+async function createStoryWithTitle(title) {
+  await createNewStory();
+  await insertStoryTitle(title);
+  await expect(page).toClick('#library-tab-text');
+  const insertButton = await page.waitForXPath(
+    `//button//span[contains(text(), 'Title 1')]`
+  );
+  await insertButton.click();
+  await expect(page).toMatchElement('[data-testid="textFrame"]', {
+    text: 'Title 1',
+  });
+}
+
+async function updateFont(fontFamily) {
+  // Open style pane
+  await expect(page).toClick('li[role="tab"]', { text: /^Style$/ });
+  await expect(page).toClick('button[aria-label="Font family"]');
+  await page.keyboard.type(fontFamily);
+  await page.keyboard.press('ArrowDown');
+  await page.keyboard.press('Enter');
+}
+
+async function addStoryWithFont(title) {
+  await createStoryWithTitle(title);
+  await updateFont('Rock Salt');
+  await publishStory();
+  // Somehow reloading the page after creating the story here helps
+  // with making the test more stable, i.e. toggling dev tools
+  // works more reliably.
+  await page.reload();
+}
+
+function isPlatformMacOS() {
+  return page.evaluate(() => {
+    const { platform } = window.navigator;
```

Comment:
Should we be using window here?

Reply:
Yes, `navigator` is a property of the `window` global.